WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/logic/ui/EventManager.java

```diff
@@ -3,16 +3,18 @@
 import java.util.Date;
 
 import com.intellij.openapi.editor.Editor;
-import nl.tudelft.watchdog.logic.document.Document;
+import nl.tudelft.watchdog.core.logic.interval.intervaltypes.*;
+import nl.tudelft.watchdog.core.logic.document.Document;
 import nl.tudelft.watchdog.logic.document.DocumentCreator;
+import nl.tudelft.watchdog.logic.document.EditorWrapper;
 import nl.tudelft.watchdog.logic.interval.IntervalManager;
 import nl.tudelft.watchdog.logic.interval.intervaltypes.*;
-import nl.tudelft.watchdog.logic.ui.events.WatchDogEvent;
-import nl.tudelft.watchdog.logic.ui.events.WatchDogEvent.EventType;
+import nl.tudelft.watchdog.core.logic.ui.events.WatchDogEvent;
+import nl.tudelft.watchdog.core.logic.ui.events.WatchDogEvent.EventType;
 
 
 /**
- * Manager for {@link nl.tudelft.watchdog.logic.ui.events.EditorEvent}s. Links such events to actions in the
+ * Manager for {@link nl.tudelft.watchdog.core.logic.ui.events.EditorEvent}s. Links such events to actions in the
```

Comment:
i have the feeling we can extract a lot functionality from here to the core, but please keep that in mind for later.